### PR TITLE
[CIAPP] Update Azure docs for extension instead of webhooks

### PR DESCRIPTION
**Pull request by AdrianLC**
### What does this PR do?

Change the docs for CI Visibility for Azure Pipelines to make use of the new extension instead of configuring webhooks directly.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/pipelines/azure.md [hunk 1]
```diff
@@ -32,17 +32,61 @@ further_reading:
 
 The Datadog integration for [Azure Pipelines][1] works by using [service hooks][2] to send data to Datadog.
```
Comment (andrea-mosk) [inline content/en/continuous_integration/pipelines/azure.md:33]: Is this still accurate?

> works by using [service hooks][2] ...

File: content/en/continuous_integration/pipelines/azure.md [hunk 2]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
 
+1. Install the [Datadog CI Visibility][8] extension from the Azure Marketplace.
+
 1. For each project, go to **Project settings > Service hooks** in Azure DevOps and select `Create subscription` (green plus icon).
```
Comment (andrea-mosk) [inline content/en/continuous_integration/pipelines/azure.md:37]: This should be 2. and so on right? I see the UI displays it well, but let's clean it here

File: content/en/continuous_integration/pipelines/azure.md [hunk 3]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
     - **Run state changed**
     - **Run stage state changed**
     - **Run job state changed**
 
 3. Click `Next` to continue to the next step and set the following:
-    - **URL**: <code>https://webhook-intake.{{< region-param key="dd_site" >}}/api/v2/webhook/</code>
-    - **HTTP Headers**: `DD-API-KEY:<API_KEY>` where `<API_KEY>` is your [Datadog API key][3].
-    - **Resource Version**: Scroll down to the latest field and choose `[Latest]`.
+    - **Datadog Site**: {{< region-param key="dd_site" >}}
+    - **Datadog API Key**: `<API_KEY>` where `<API_KEY>` is your [Datadog API key][3].
```
Comment (andrea-mosk) [inline content/en/continuous_integration/pipelines/azure.md:46]:
```suggestion
    - **Datadog API Key**: your [Datadog API key][3].
```

Maybe?

File: content/en/continuous_integration/pipelines/azure.md (hunk 1 above)
Comment (AdrianLC) [reply content/en/continuous_integration/pipelines/azure.md:33]: yes the extension is still service hooks

**Review by andrea-mosk — APPROVED** [review]

File: content/en/continuous_integration/pipelines/azure.md [hunk 4]
```diff
@@ -32,17 +32,61 @@ further_reading:
 
 The Datadog integration for [Azure Pipelines][1] works by using [service hooks][2] to send data to Datadog.
 
-1. For each project, go to **Project settings > Service hooks** in Azure DevOps and select `Create subscription` (green plus icon).
+1. Install the [Datadog CI Visibility][8] extension from the Azure Marketplace.
 
-2. Create a new subscription with the same URL for each of the following webhook types:
+2. For each project, go to **Project settings > Service hooks** in Azure DevOps and select `Create subscription` (green plus icon).
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:37]:
```suggestion
2. For each project, go to **Project settings > Service hooks** in Azure DevOps and select the green plus (+) icon to create a subscription.
```

File: content/en/continuous_integration/pipelines/azure.md [hunk 5]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
+3. Create a new subscription to the now available service `Datadog CI Visibility` for each of the following webhook types:
     - **Run state changed**
     - **Run stage state changed**
     - **Run job state changed**
 
-3. Click `Next` to continue to the next step and set the following:
-    - **URL**: <code>https://webhook-intake.{{< region-param key="dd_site" >}}/api/v2/webhook/</code>
-    - **HTTP Headers**: `DD-API-KEY:<API_KEY>` where `<API_KEY>` is your [Datadog API key][3].
-    - **Resource Version**: Scroll down to the latest field and choose `[Latest]`.
+4. Click `Next` to continue to the next step and set the following:
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:44]:
```suggestion
4. Click **Next** to continue to the next step and set the following:
```

File: content/en/continuous_integration/pipelines/azure.md [hunk 6]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
+    - **Datadog Site**: {{< region-param key="dd_site" >}}
+    - **Datadog API Key**: your [Datadog API key][3].
+
+5. Click `Finish`.
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:48]:
```suggestion
5. Click **Finish**.
```

File: content/en/continuous_integration/pipelines/azure.md [hunk 7]
```diff
@@ -32,17 +32,61 @@ further_reading:
 
 The Datadog integration for [Azure Pipelines][1] works by using [service hooks][2] to send data to Datadog.
 
-1. For each project, go to **Project settings > Service hooks** in Azure DevOps and select `Create subscription` (green plus icon).
+1. Install the [Datadog CI Visibility][8] extension from the Azure Marketplace.
 
-2. Create a new subscription with the same URL for each of the following webhook types:
+2. For each project, go to **Project settings > Service hooks** in Azure DevOps and select `Create subscription` (green plus icon).
+
+3. Create a new subscription to the now available service `Datadog CI Visibility` for each of the following webhook types:
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:39]:
```suggestion
3. Create a new subscription to the `Datadog CI Visibility` service for each of the following webhook types:
```

File: content/en/continuous_integration/pipelines/azure.md [hunk 8]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
-    - **Resource Version**: Scroll down to the latest field and choose `[Latest]`.
+4. Click `Next` to continue to the next step and set the following:
+    - **Datadog Site**: {{< region-param key="dd_site" >}}
+    - **Datadog API Key**: your [Datadog API key][3].
+
+5. Click `Finish`.
+
+<div class="alert alert-info">
+All 3 supported types of events are required and must be enabled individually.
+If one or more events are not enabled, the installation will be incomplete and lead to unexpected behavior in Datadog.
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:52]:
```suggestion
All 3 supported types of events are required and must be enabled individually.
Not enabling one or more events results in an an incomplete installation, leading to unexpected behavior in Datadog.
```

File: content/en/continuous_integration/pipelines/azure.md [hunk 9]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
+</div>
+
+### Configuring multiple projects in bulk
+
+
+If you want to enable the hooks for many or all your Azure projects we provide a [script](https://raw.githubusercontent.com/DataDog/ci-visibility-azure-pipelines/main/service_hooks.py) to help you do it through the Azure API. For this you'll need:
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:58]:
```suggestion
If you want to enable the hooks for many or all your Azure projects, Datadog provides a [script](https://raw.githubusercontent.com/DataDog/ci-visibility-azure-pipelines/main/service_hooks.py) to help you do it through the Azure API. 

To run the script, you need:
```

File: content/en/continuous_integration/pipelines/azure.md [hunk 10]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
+
+- Azure DevOps username
+- Azure DevOps [API Token](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows#create-a-pat)
+- Azure DevOps organization name
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:62]:
```suggestion
- An Azure DevOps username
- An Azure DevOps [API Token](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows#create-a-pat)
- An Azure DevOps organization name
```

File: content/en/continuous_integration/pipelines/azure.md [hunk 11]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
+
+The script only needs python3 and the requests package. For more info:
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:64]:
```suggestion
The script only needs python3 and the requests package. For more info, run:
```

File: content/en/continuous_integration/pipelines/azure.md [hunk 12]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
+```
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:65]:
~~~suggestion
```shell
~~~

File: content/en/continuous_integration/pipelines/azure.md [hunk 13]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
+./service_hooks.py --help
+```
+
+The script supports environment variables `DD_API_KEY` and `DD_SITE` and also flag parameters `--dd-api-key` and `--dd-site`
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:69]:
```suggestion
The script supports environment variables `DD_API_KEY` and `DD_SITE`, and flag parameters `--dd-api-key` and `--dd-site`.
```

File: content/en/continuous_integration/pipelines/azure.md [hunk 14]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
+
+Example to enable for all the projects:
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:71]:
```suggestion
Example for enabling the hook in all projects:
```

File: content/en/continuous_integration/pipelines/azure.md [hunk 15]
```diff
@@ -32,17 +32,61 @@ further_reading:
...
+```
+./service_hooks.py \
+    --dd-api-key ******************** \
+    --az-user "John Doe" \
+    --az-token ********************** \
+    --az-org datadoghq \
+    --threads 4
+```
+
+Example to enable only in a couple of projects:
```
Comment (hestonhoffman) [inline content/en/continuous_integration/pipelines/azure.md:81]:
```suggestion
Example for enabling the hook in specified projects:
```

**Review by hestonhoffman — CHANGES_REQUESTED** [review]

File: content/en/continuous_integration/pipelines/azure.md (hunk 2 above)
Comment (AdrianLC) [reply content/en/continuous_integration/pipelines/azure.md]: The number was off but the rest are not 1

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**